src/util/filter-toolbox.js

```diff
@@ -39,6 +39,7 @@ var filterToolbox = function (toolbox, opcodes) {
             if (category.nodeName.toLowerCase() !== 'category') continue;
             var filteredCategory = filterToolboxNode(category, opcodes);
             if (filteredCategory.hasChildNodes()) filteredToolbox.appendChild(filteredCategory);
+            if (!filteredCategory.hasChildNodes() && filteredCategory.hasAttribute("custom")) filteredToolbox.appendChild(filteredCategory);
```

Comment:
Hey @TheBrokenRail — the tests are failing because this line is too long and the string is quoted with double quotes. Would you please make this a multi-line if statement and change the quote marks to single quotes?